backend/danswer/datastores/datastore_utils.py

```diff
@@ -30,3 +32,13 @@ def get_uuid_from_chunk(
         [doc_str, str(chunk.chunk_id), str(mini_chunk_ind)]
     )
     return uuid.uuid5(uuid.NAMESPACE_X500, unique_identifier_string)
+
+
+def ensure_doc_updated_time(t: datetime | None) -> int | None:
```

Comment:
nit: add method header/rename - it's not immediately clear what this does based on the name

Reply:
my naming makes perfect sense, idgi
(fine I'll rename it)